Enh/#1/new branch

Prajwal-tg opened the pull request.

Prajwal-tg committed:
Initial commit

Prajwal-tg committed:
Updated README file

Prajwal-tg committed:
updated README #2

Prajwal-tg committed:
Added python output screenshot

Prajwal-tg committed:
Updated README file #3

Prajwal-tg committed:
Updated README.md file

Prajwal-tg committed:
Updated README.md file

Sandeep-George reviewed (changes requested):
DOC: Why is there multiple Readme, you only need to update the original readme
DOC: The syntax highlighting is absent
ENV : The gitignore should have datasets and the Zone Identifier files 
ENV : The Screenshot should not be uploaded to github as a commit, Instead share it in the conversation space where you request PR.

Prajwal-tg committed:
Suggested changes are done

Prajwal-tg committed:
Updated gitignore file

Prajwal-tg committed:
Updated README file

Prajwal-tg commented:
Python code output(screenshot)
![Screenshot (17)](https://user-images.githubusercontent.com/110401129/187853158-cb9f3702-3791-428f-9cf3-fca320f2dfd3.png)

Sandeep-George reviewed (changes requested):
Your branch has merge conflicts - please resolve it first

Prajwal-tg commented:
I have deleted the .gitignore file from the main branch as it was stated that there should be only one empty README file in main branch, still i got merge conflicts issue. now i have added new .gitignore file in the main branch to resolve the merge conflict. Is this the correct way to do it. And also in (Enh/#1?new branch) branch there is a README.md file and there is empty README file in main branch, when we merge two branches, then we would have two readme files(README and README.md) in the main branch?
